frontend/src/scenes/data-pipelines/batch-exports/BatchExportEditForm.tsx

```diff
@@ -493,6 +494,133 @@ export function BatchExportsEditFields({
                             </LemonField>
                         ) : null}
                     </>
+                ) : batchExportConfigForm.destination === 'Databricks' ? (
+                    <>
+                        <LemonField name="integration_id" label="Integration">
+                            {({ value, onChange }) => (
+                                <IntegrationChoice integration="databricks" value={value} onChange={onChange} />
+                            )}
+                        </LemonField>
+
+                        <LemonField
+                            name="http_path"
+                            label="HTTP Path"
+                            info={<>HTTP Path value for your all-purpose compute or SQL warehouse.</>}
+                        >
+                            <LemonInput placeholder="/sql/1.0/warehouses/my-warehouse" />
```

Comment:
Could we simplify this parameter somehow? For example, is the `/sql/1.0/warehouses/` prefix always the same? If so, we could just exclude it.

Reply:
I can also see a common mistake that users paste the whole URL instead of just the path.